sorc/build_tracker.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+set -xeu
+source ./machine-setup.sh > /dev/null 2>&1
+
+#Supports Debug or Release modes for the build
+BUILD_MODE=${BUILD_MODE:-Release}
+
+cwd=$(pwd)
+
+if [ "${BUILD_MODE}" = Release ]; then
+  export BUILD_TYPE=RELEASE
+else
+  export BUILD_TYPE=DEBUG
+fi
+
+module use ../modulefiles
+module load $target.lua
+module list
+
+cd ..
+
+if [ -d "build" ]; then
+   rm -rf build
+fi
+mkdir build
+cd build
+
+cmake .. -DCMAKE_Fortran_COMPILER=${CMAKE_Fortran_COMPILER} -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER} -DCMAKE_BUILD_TYPE=${BUILD_TYPE}
+
+make -j 8 VERBOSE=2
```

Comment:
Again, another time, but it may be useful to have BUILD_JOBS and VERBOSE options set as variables similar to BUILD_MODE above so the user can specify how verbose they want it to build.